execution-order/execution-order.js

```diff
@@ -0,0 +1,64 @@
+/* global DraggableList */
+/* global prefs */
+/* global API */
+'use strict';
+
+(async () => {
+  const list = (await getOrderedStyles()).map(createLi);
+  const ol = document.querySelector('#style-list');
+  let maxTranslateY;
+  ol.append(...list.map(l => l.el));
+  ol.addEventListener('d:dragstart', e => {
+    e.detail.origin.dataTransfer.setDragImage(new Image(), 0, 0);
+    maxTranslateY = ol.scrollHeight - e.detail.dragTarget.offsetHeight - e.detail.dragTarget.offsetTop;
+  });
+  ol.addEventListener('d:dragmove', e => {
+    e.detail.origin.dataTransfer.dropEffect = 'move';
+    const y = Math.min(e.detail.currentPos.y - e.detail.startPos.y, maxTranslateY);
+    e.detail.dragTarget.style.transform = `translateY(${y}px)`;
+  });
+  ol.addEventListener('d:dragend', e => {
+    const [item] = list.splice(e.detail.originalIndex, 1);
+    list.splice(e.detail.spliceIndex, 0, item);
+    ol.insertBefore(e.detail.dragTarget, e.detail.insertBefore);
+    prefs.set('styles.order', list.map(l => l.style._id));
+  });
+  new DraggableList(ol, {scrollContainer: ol});
+  document.querySelector('#main').classList.add('ready');
+
```

Comment:
What is the purpose of this empty line as well as L56?

Reply:
I guess it is to group the code that setups the list.

Reply:
It seems random to me, I don't see anything special in the separated block.